cmd/crates/soroban-spec-typescript/src/project_template/src/invoke.ts

```diff
@@ -41,22 +31,40 @@ async function getAccount(): Promise<Account | null> {
 
 export class NotImplementedError extends Error { }
 
+type Simulation = SorobanClient.SorobanRpc.SimulateTransactionResponse
```

Comment:
wait, how does this work if `SorobanClient` isn't default-imported?

Reply:
ideally we would avoid the default-import to keep bundles slimmer (i.e. only import what we need)

Reply:
Great point! Let's do that in a separate PR, though, since this one only maintained the existing import structure

Reply:
+1 always hated default imports.

Reply:
@Shaptic great point! Let's do this in a follow-up